src/core/declaration.ts

```diff
@@ -114,15 +114,24 @@ export function getDeclaration(ctx: Context, filepath: string, originalImports?:
     directive: stringifyDeclarationImports({ ...originalImports?.directive, ...imports.directive }),
   }
 
-  let code = `// generated by unplugin-vue-components
-// We suggest you to commit this file into source control
-// Read more: https://github.com/vuejs/core/pull/3399
+  const isVue27 = ctx.options.vueVersion ? ctx.options.vueVersion.startsWith('2.7') : false
+  let dtsHead = ''
+  isVue27
+    ? dtsHead = `  
+export {}
+        
+declare module 'vue' {`
+    : dtsHead = `
```

Comment:
Avoid repeat assignment

```suggestion
  const isVue27 = ctx.options.vueVersion ? ctx.options.vueVersion.startsWith('2.7') : false
  const dtsHead = isVue27
    ? `  
export {}
        
declare module 'vue' {`
    : `
```